modules/shortcodes/js/blocks/vr-block.jsx

```diff
@@ -53,35 +54,36 @@
 					<div>
 						<Placeholder
 							key="placeholder"
-							instructions={ __( 'Enter URL to VR image', 'jetpack' ) }
 							icon="format-image"
 							label={ __( 'VR Image', 'jetpack' ) }
 							className={ props.className }
 						>
-							<UrlInput
+							<TextControl
+								type="url"
+								style={ { flex: '1 1 auto' } }
```

Comment:
I'm not sure this `flex: '1 1 auto'` is really necessary here - could you expand on why we need it? On a quick test, when I remove it I don't see a difference.